### PR TITLE
Dpp 1176 provider transfer final approver ui notification

### PR DESCRIPTION
**Pull request by chrisfoster76**

### CODE REVIEW

**Comment by chrisfoster76** [comment]
Please ignore the bulk upload and command handler refactoring changes, they are not part of this change and I'm not clear on why they show up in the diff, since they are already in the transfersbasebranch.

File: src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Orchestrators/Commitments/WhenGettingApprovedViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        {
+            _commitment.TransferSenderId = isTransfer ? 1L : default(long?);
+
+            var result = await _orchestrator.GetApprovedViewModel(1, "Hashed-Id");
+
+            Assert.AreEqual(expectHeadline, result.Headline);
+        }
+
+        [Test]
+        public async Task ThenTheViewModelShouldIndicateWhenCohortIsForTransfer()
```
Comment (lazcool) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Orchestrators/Commitments/WhenGettingApprovedViewModel.cs]: These 2 tests could use [TestCase].

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Commitment/RequestApproved.cshtml [hunk 2]
```diff
@@ -36,14 +36,33 @@
...
-            <a href="@Model.RedirectUrl" aria-label="@Model.RedirectLinkText">@Model.RedirectLinkText</a>
+            <p id="changeMainCopy">
+                Your cohort has been approved. To update the details for any apprentices in this cohort you will need to
+                <a href="@Url.Action("Index", "ManageApprentices")">manage your apprentices</a>.
+            </p>
+        }
+        
+        @if (Model.HasOtherCohortsAwaitingApproval)
+        {
+            <a href="@Url.Action("ReadyForReview")">Return to Approve cohorts</a>
```
Comment (lazcool) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Commitment/RequestApproved.cshtml]: As discussed, these Url's when generated in CommitmentController used to supply the provider id (and the actions seem to require them). Do we need to supply the provider id here?

Also, is the Url.Action clever enough to pick up the action is in the commitment controller (now that the Url generation has moved into the view).

**Comment by lazcool** [comment]
Reviewed the following files for this PR (ignored bulk upload and command handler refactoring changes as requested)...

WhenGettingAcknowledgementViewModel.cs
WhenGettingApprovedViewModel.cs
CommitmentController.cs
ApprovedViewModel.cs
CommitmentDetailsViewModel.cs
CommitmentOrchestrator.cs
RequestApproved.cshtml

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Commitment/RequestApproved.cshtml (hunk 2 above)
Comment (lazcool) [reply src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Commitment/RequestApproved.cshtml]: Chris has shown this works. Looks like Url.Action uses the route values from the current request...
https://stackoverflow.com/questions/7133223/asp-net-mvc-url-action-adds-current-route-values-to-generated-url

**Pull request merged**